Shokofin/Providers/Helper.cs

```diff
@@ -4,7 +4,10 @@
 using System.Text.RegularExpressions;
 using Shokofin.API.Models;
 using Title = Shokofin.API.Models.Title;
-using DisplayTitleType = Shokofin.Configuration.PluginConfiguration.DisplayTitleType;
+using DisplayLanguageType = Shokofin.Configuration.PluginConfiguration.DisplayLanguageType;
+using EpisodeType = Shokofin.API.Models.Episode.EpisodeType;
+using Models = Shokofin.API.Models;
```

Comment:
`Series`, `Episode.AniDB` work just fine for me without this as `Models` is already imported at line 5. Was there a problem with that?

Reply:
It was left over from moving the code between files.